traffic_control/clients/python/trafficops/tosession.py

```diff
@@ -866,6 +866,17 @@ def assign_deliveryservice_servers_by_names(self, xml_id=None, data=None):
 		:raises: Union[LoginError, OperationError]
 		"""
 
+	@api_request(u'post', u'deliveryservice_requests', ('2.0',))
```

Comment:
You don't need those `u`s in there, they don't do anything.

Reply:
This change probably belongs in its own PR IMO -- it has nothing to do with "Document undocumented Traffic Ops routes".

Reply:
Yeah, that's fair too.

Reply:
Okay, removed in 91194ac018. The only real change to the Python client for TO is the route and docs reference rename in eaeeba7fce to avoid the "undefined label" warning that you would otherwise get building the docs:

```
/home/zrhoffman/go/src/github.com/apache/trafficcontrol/traffic_control/clients/python/trafficops/tosession.py:docstring of trafficops.TOSession.create_cdn_dns_sec_keys:1: WARNING: undefined label: to-api-deliveryservices-dnsseckeys-generate (if the link has no caption the label must precede a section header)
```